README.md

```diff
@@ -64,9 +64,15 @@ access_token: "{{ lookup('env', 'PERSONAL_ACCESS_TOKEN') }}"
 # Is it the runner for organization or not
 runner_org: no
 
-# GitHub Repository user or Organization owner used for Runner registration
+# Name to assign to this runner in GitHub
+runner_name: "{{ lookup('pipe', 'hostname') }}"
```

Comment:
Hi there. I just realized you used`lookup('pipe', 'hostname')"`. This is returning the hostname of the control node from where you are running the Ansible playbook. This may be fine if you are running it locally, but not good if you need to name the runner by a remote/destination server name (or installing more than one runner). So i will use ansible facts for this. The line will look like that:
```
runner_name: "{{ ansible_hostname }}"
```

Reply:
Good catch, thanks!